.github/workflows/workflow-bytebase-sql-review.md

```diff
@@ -0,0 +1,16 @@
+# Bytebase SQL review
+
+## Overview
+
+This workflow is used to review SQL changes in Bytebase. It is triggered by a
+pull request to the `main` branch of the calling repository.
+
+## Usage
+
+- **Purpose:** Parse and check the SQL statement according to the SQL review
+  rules set in our Bytebase instance to detect SQL anti-patterns and enforce
+  schema consistency across the organization.
+- **Secrets:** Secrets need to be configured from the calling repository.
+  - `BYTEBASE_API`: The API key used to authenticate with Bytebase.
```

Comment:
We might need to be a little bit more specific for the origin of the API token

Reply:
Done